schemas/styles/csl.rnc

```diff
@@ -649,6 +650,72 @@ div {
 }
 # ==============================================================================
 
+## cs:title Rendering Element
+div {
+  ## Use to render titles if "title-main" and "title-sub" need different formatting
+  ## or to omit either "title-main" or "title-sub"
+  rendering-element.title =
+    element cs:title {
+      title.attributes,
+      title.formatting,
+      (title.main?, title.sub?)
+    }
+
+  title.attributes =
```

Comment:
I'm not sure if the diffs are just getting screwed up, but I can't tell what's really new. if you haven't run trang to update the formatting, please do that and I'll take another look and see if it's easier to see.

Reply:
Ok. have done.